src/index.css

```diff
@@ -35,29 +57,136 @@ ul {
 
 .books-list {
   display: flex;
-  width: 250px;
-  justify-content: space-between;
+  height: 7rem;
+  width: 70%;
+  background-color: #fff;
+  padding: 5px 1.688rem;
+  margin-left: 4.25rem;
+}
+
```

Comment:
- [ ] Good job so far 👏 I would suggest that you improve your styles in order to make the book components align in the center of the page. You can check out what I mean by looking at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/201370860-27876488-62c0-4aea-9c62-370413ad1239.png)

**Zeplin design**
![image](https://user-images.githubusercontent.com/90222110/201371019-eb77fb35-8390-4822-a167-59963eed3a56.png)

Please stick to the design given as much as possible 😃 👍